flake.nix

```diff
@@ -95,7 +95,7 @@
 
             sharedAttrs = {
               pname = "riff";
-              version = "0.0.0-unreleased";
+              version = "1.0.0";
               src = self;
```

Comment:
It occurs to me these details could be loaded with the builtin toml reader?

Reply:
this is a random thought, not a thing.

Reply:
It's a good idea. Done.